dockerfiles/Dockerfile.openvino

```diff
@@ -5,7 +5,7 @@
 
 FROM ubuntu:18.04
 
-ARG DEVICE=CPU_FP32
+ARG DEVICE=MYRIAD_FP16
```

Comment:
Why the default value is changed?

Reply:
This was a request from Manash and George as they observed that that's the configuration that most people are using according to analytics

Reply:
yes, the goal is to publish single image instead of the current 4, and default to myriad. (others can be switched to at run-time) but I think that can be done independently of Dockerfile modification. (i.e. the pipeline which builds/publishes the image can be invoked with MYRIAD argument)
For this Dockerfile, on second thought it may be better to leave it at CPU_FP32 because that is guaranteed to work (whereas not everyone has a MYRIAD device)